configs/scripts/program/build.sh

```
@@ -1,142 +1,41 @@
-    # so the resulting .so hash matches a remote verification of the same
-    # source. Bubblegum's Cargo workspace lives under programs/bubblegum, so the
-    # build must run from that directory instead of the repository root.
-    (
-        cd "${WORKSPACE_DIR}"
-        solana-verify build "${BASE_IMAGE_ARGS[@]}" --library-name "${LIB_NAME}" -- --package "${PACKAGE_NAME}" ${ARGS[@]+"${ARGS[@]}"}
-    )
+WORKING_DIR=$(pwd)
+export SBF_OUT_DIR="${WORKING_DIR}/${OUTPUT}"
 
-    cp "${WORKSPACE_DIR}/target/deploy/${LIB_NAME}.so" "${WORKING_DIR}/${OUTPUT}/${LIB_NAME}.so"
+for p in ${PROGRAMS[@]}; do
```

Shellcheck (0.11.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)